src/coreclr/jit/codegenxarch.cpp

```diff
@@ -10894,9 +10894,12 @@ void CodeGen::genZeroInitFrameUsingBlockInit(int untrLclHi, int untrLclLo, regNu
         assert((blkSize + alignmentHiBlkSize) == (untrLclHi - untrLclLo));
 #endif // !defined(TARGET_AMD64)
 
+        const int maxSimdSize = (int)compiler->roundDownSIMDSize(blkSize);
```

Comment:
This is already accounting for preferred simd width, right? Just want to ensure we don't generate zmm based instructions on 1st gen capable hardware by default.

Reply:
Yep, `roundDownSIMDSize` uses preffered vector width logic